decred/decred/util/helpers.py

```diff
@@ -21,6 +21,19 @@
 from appdirs import AppDirs
 
 
+def satsToCoins(sats):
```

Comment:
Despite the name of the `UTXO` field, which was copied from somewhere, we don't have Satoshis in Decred. We have atoms.